redis/reply.go

```diff
@@ -394,13 +394,16 @@ func Int64Map(result interface{}, err error) (map[string]int64, error) {
 
 // Positions is a helper that converts an array of positions (lat, long)
 // into a [][2]float64. The GEOPOS command returns replies in this format.
-func Positions(result interface{}, err error) ([][2]float64, error) {
+func Positions(result interface{}, err error) ([]*[2]float64, error) {
 	values, err := Values(result, err)
 	if err != nil {
 		return nil, err
 	}
-	positions := make([][2]float64, len(values))
+	positions := make([]*[2]float64, len(values))
 	for i := range values {
+		if values[i] == nil {
+			continue
+		}
 		p, ok := values[i].([]interface{})
 		if !ok {
```

Comment:
If the position for a given input member is returned in the response,do we have to even check if the position contains both latitude and longitude ? Does geoadd command allow not specifying either latitude or longitude for a given member.